tests/NHSD.BuyingCatalogue.Ordering.Api.UnitTests/Builders/CreateOrderItemModelBuilder.cs

```diff
@@ -39,10 +39,16 @@ private CreateOrderItemModelBuilder()
             _price = 25.1m;
         }
 
+        public CreateOrderItemModelBuilder WithQuantity(int? quantity)
+        {
+            _quantity = quantity;
+            return this;
+        }
+
         public static CreateOrderItemModelBuilder Create() => new CreateOrderItemModelBuilder();
 
-        public CreateOrderItemModel Build() =>
-            new CreateOrderItemModel
+        public CreateOrderItemSolutionModel Build() =>
```

Comment:
I thought about this, and I think we can reuse the builder for all 3 order item types. My suggestion is renaming Build to BuildSolution, and then we can reuse it for the other 2 types instead of having 3 builders that all look essentially the same.